src/components/User.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+function User(props) {
+	const user = props;
```

Comment:
Here you are not using destructuring, so if you had more information it would throw an error:
```
const {user} = props;
```